PSReadLine/Cmdlets.cs

```diff
@@ -136,7 +136,9 @@ public PSConsoleReadLineOptions(string hostName)
             ContinuationPromptColor = Console.ForegroundColor;
             ExtraPromptLineCount = DefaultExtraPromptLineCount;
             AddToHistoryHandler = null;
+            DetectSensitiveInputHandler = null;
             HistoryNoDuplicates = DefaultHistoryNoDuplicates;
+            ScrubSensitiveHistory = DefaultScrubSensitiveHistory;
```

Comment:
Are 2 options really necessary. I worry when a tool has too many configuration options and really tried to avoid unnecessary configuration options to PSReadLine.

Reply:
Well, I guess I can just leave the `DetectSensitiveInputHandler` option and make our regex pattern logic as the default handler. If user set it to `null`, then meaning they want to disable `ScrubSensitiveHistory` functionality.
How does this sound?

Reply:
Maybe we don't need a new option at all - instead creating a default `AddToHistoryHandler`.

I guess I'm not sure it's useful to keep sensitive command lines in memory but not on disk. After all, password managers clear the clipboard after a short period.

Reply:
PR updated to use `AddToHistoryHandler`. Changed its type to `Func<string, object>`.